fix(css): use better terms for statement and block @layers (#34991)
mdn/content: files/en-us/web/css/@layer/index.md

The declaration order matters. The first declared layer gets the lowest priority and the last declared layer gets the highest priority. However, the priority is reversed when the [`!important`](/en-US/docs/Web/CSS/important) flag is used.

> The `@layer` at-rule is used to create an inline cascade layer in one of three ways.

Author: ```suggestion
The `@layer` at-rule is used to create a cascade layer in one of three ways.
```
